spec/app_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+require "rack/test"
+require_relative '../../app'
+
+describe Application do
+  # This is so we can use rack-test helper methods.
+  # hello 
+  include Rack::Test::Methods
+
+  # We need to declare the `app` value by instantiating the Application
+  # class so our tests work.
+  let(:app) { Application.new }
+
+  # run seed files and start database connection 
+  before(:each) do 
+    chitter_seeds = File.read('spec/seeds/chitter_seeds.sql')
```

Comment:
Can be nice to break these 3 lines out into a method so that it's clearer that they achieve a single goal

By having a method name like replace_tables it's very clear that before each test you are resetting the tables